packages/cli/src/dirCommand.js

```diff
@@ -50,7 +50,10 @@ async function dirCommand(
     if (!isCompilable(relative)) return false
     relative = rename(relative, ext, filenameCase)
 
-    const dest = path.join(program.outDir, relative)
+    const dest = program.outDir.startsWith('/')
```

Comment:
Why not always using `path.resolve()`?

Reply:
You're right. I've removed the condition and always use `path.resolve()` here.